src/basic/FStarC.Common.fst

```diff
@@ -87,8 +82,8 @@ let __string_of_list (delim:string) (f : 'a -> string) (l : list 'a) : string =
 (* Why two? This function was added during a refactoring, and
 both variants existed. We cannot simply move to ";" since that is a
 breaking change to anything that parses F* source code (like Vale). *)
-let string_of_list = __string_of_list ", "
-let string_of_list' = __string_of_list "; "
+let string_of_list  f l = __string_of_list ", " f l
```

Comment:
@mtzguido Do you know why this removes the unit argument in extraction?  This seems like a really innocuous change.